roles/core/repositories_client/tasks/centos_7.yml

```diff
@@ -7,3 +7,8 @@
     baseurl: "http://{{networks[j2_node_main_network]['services_ip']['repository_ip']}}/repositories/{{equipment_profile['operating_system']['distribution']}}/{{equipment_profile['operating_system']['distribution_version']}}/{{equipment_profile['hardware']['cpu']['architecture']}}/{{item}}"
     gpgcheck: no
   with_items: "{{ repositories }}"
+  
+- name: Remove repository
```

Comment:
Remove any previous repository *before* creating new ones. If a user create a repo with prefix centOS, this change will remove it.